libs/designsystem/src/lib/components/chart-wip/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { all as deepMergeAll, Options as deepMergeOptions } from 'deepmerge';
+
+export function deepMergeObjects(...objects: any[]) {
+  const overwriteMerge = (_target: any[], source: any[], _options: deepMergeOptions) => source;
+  const deepMergeOptions: deepMergeOptions = {
+    arrayMerge: overwriteMerge,
+  };
+
+  const objectsWithoutUndefined = objects.filter((object) => object !== undefined);
+
+  return deepMergeAll([{}, ...objectsWithoutUndefined], deepMergeOptions);
+}
```

Comment:
Is there a more appropriate place to store this function? It is not necessarily only useful in the context of this component.